src/components/Loader/Loader.jsx

```diff
@@ -0,0 +1,9 @@
+export const Loader = () => (
+  <div className='loader'>
+    <div className='loader__elem' />
+    <div className='loader__elem' />
+    <div className='loader__elem' />
+    <div className='loader__elem' />
+    <div className='loader__elem' />
```

Comment:
it's better to create an array and generate a key for each elem

Reply:
thx, i changed this element. 
```suggestion
  {loaderElementsList.map((item) => (
      <div key={item} className='loader__elem' />
    ))}
```